examples/Bus3Phasor/README.md

```diff
@@ -0,0 +1,147 @@
+Each component is sub-index by the components ID. The 3Bus model is composed of 3 buses, 3 branches, 2 loads and 2 generators. For simplicity in some equation, $I = \left[\begin{array}{c} I_r \\ I_i \end{array}\right]$ to represent complex variables.
+
+## Bus equations:
+```math 
+I_{G,1}+I_{2,1}+I_{3,1}+I_{L,1} = 0
+```
+```math
+I_{L,2}+I_{1,2}+I_{3,2} = 0
+```
+```math 
+I_{G,3}+I_{1,3}+I_{2,3} = 0
+```
+
+## Branch equations:
+
+(branch connecting Bus 1 and Bus 2)
+```math
+\left[ \begin{array}{c}
+        I_{2,1} \\
+        I_{1,2}
+    \end{array}\right] = 
+    Y^{[1]}\left[ \begin{array}{c}
+        V_1 \\
+        V_2
+\end{array}\right]
+```
+
+(branch connecting Bus 2 and Bus 3)
+```math
+\left[ \begin{array}{c}
+        I_{3,2} \\
+        I_{2,3}
+    \end{array}\right] = 
+    Y^{[2]}\left[ \begin{array}{c}
+        V_2 \\
+        V_3
+    \end{array}\right]
+```
+
+(branch connecting Bus 1 and Bus 3)
+```math
+\left[ \begin{array}{c}
+        I_{3,1} \\
+        I_{1,3}
+    \end{array}\right] = 
+    Y^{[3]}\left[ \begin{array}{c}
+        V_1 \\
+        V_3
+    \end{array}\right]
+```
+
+ ## Load equations:
+ Loads connected to bus 1:
+```math
+ I_{L,1} = \frac{V_1}{R_1} 
+``` 
+Loads connected to bus 2:
+```math
+    I_{L,2} = \frac{V_2}{R_2}
+```
+
+## Generator 1
+Generator connected to bus 1:<br>
+Differential equations: 
+```math
+\dot{\delta}_1 = \omega_1 \cdot \omega_0 
+```
+```math
+\dot{\omega}_1 = \frac{1}{2H_1}\left( \frac{P_{mech,1} - D_1\omega_1}{1+\omega_1} - T_{elec,1}\right)
+```
+```math
+\dot{E}_{qp,1} = \frac{1}{T_{dop,1}}\left(  E_{fd,1} - \left( E_{qp,1} + X_{d1,1}\left( I_{d,1} + X_{d3,1}\left( E_{qp,1} - \psi_{dp,1} - X_{d2,1}I_{d,1} \right)  \right) + \psi_{dpp,1}\cdot k_{sat,1} \right) \right)
+```
+```math 
+\dot{\psi}_{dp,1} = \frac{1}{T_{dopp,1}} \left( E_{qp,1} - \psi_{dp,1} - X_{d2,1} I_{d, 1} \right)
+``` 
+```math
+\dot{\psi}_{qp,1} = \frac{1}{T_{qopp,1}} \left( E_{dp,1} - \psi_{qp,1} + X_{q2,1}I_{q,1}\right)
+```
+```math
+\dot{E}_{dp,1} = \frac{1}{T_{qop,1}}\left(-E_{dp,1} + X_{qd,1}\psi_{qpp,1}\cdot k_{sat,1} + X_{q1,1} \left(I_{q,1} - X_{q3,1} \left(E_{dp,1} + I_{q,1}X_{q2,1} - \psi_{qp,1}\right)\right)\right)
+```
+Algebraic equations:
+```math
+V_{d,1} = -\psi_{qpp,1}(\psi_{qp^{\prime},1}E_{dp,1})(1+\omega_1)
+```
+```math
+V_{q,1} = \psi_{dpp,1}(\psi_{dp^{\prime},1}E_{qp,1})(1+\omega_1)
+```
+```math
+I_{d,1} = I_{G,1, r}\sin(\delta_1) - I_{G,1,i}\cos(\delta_1)
+```
+```math
+I_{q,1} = I_{G,1, r}\cos(\delta_1) + I_{G,1,i}\sin(\delta_1)
+```
+
+Network interfaces:
+```math
+V_{d,1} = V_{1,r}\sin(\delta_1) - V_{1,i}\cos(\delta_1) + I_{d,1} R_{a,1}  -I_{q,1} X_{qpp,1}
+```
+```math
+V_{q,1} = V_{1,r}\cos(\delta_1) + V_{1,i}\sin(\delta_1) + I_{d,1}X_{qpp,1} + I_{q,1}R_{a,1}
+```
+## Generator 3
+Generator connected to bus 3:<br>
```

Comment:
Generators may have their own labels that do not coincide with bus labels. For simplicity, in this case, you can switch bus 2 and 3 labels.